src/serviceWorker/document.ts

```diff
@@ -22,12 +20,6 @@ export function matchDocument({ request, url }: RouteMatchCallbackOptions) {
     return false
   }
 
-  // If this isn't app.uniswap.org (or a local build), skip.
-  // IPFS gateways may not have domain separation, so they cannot use document caching.
-  if (url.hostname !== 'app.uniswap.org' && !isDevelopment()) {
```

Comment:
Is there a need for us to do something similar or can we skip this?

Reply:
I think it's redundant, I don't see nothing bad to use service-worker on the IPFS deployment or elsewhere